client/extensions/woocommerce/app/product-categories/form.js

```diff
@@ -216,6 +260,34 @@ class ProductCategoryForm extends Component {
 								<FormLabel htmlFor="description">{ translate( 'Description' ) }</FormLabel>
 								{ this.renderTinyMCE() }
 							</FormFieldSet>
+							<FormFieldSet>
+								<FormLabel>
+									<FormCheckbox
+										checked={ this.state.isTopLevel }
+										onChange={ this.onTopLevelChange }
+									/>
+									<span>{ translate( 'Top level category' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Top level category', { context: 'Categories: New category being created is top level i.e. has no parent'} )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-product-cat-parent-field)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).